CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
+
+* [#118](https://github.com/rubocop-hq/rubocop-minitest/pull/118): Fix `Minitest/AssertEmptyLiteral` by making it check for `assert_equal([], array)` instead of `assert([], array)`. ([@cstyles][])
```

Comment:
This is a breaking change. Can you mark `**(Breaking)**` and squash your commits into one?

```suggestion
* [#118](https://github.com/rubocop-hq/rubocop-minitest/pull/118): **(Breaking)** Fix `Minitest/AssertEmptyLiteral` by making it check for `assert_equal([], array)` instead of `assert([], array)`. ([@cstyles][])
```

Reply:
Fixed in 70f02e09b89d980c67a0c60af80778411fe12b90